docs/walk-through/parameters.md

```diff
@@ -136,9 +179,8 @@ def repeat_back(message: str):
     print(f"You just said: '{message}'")
 ```
 
-Let's say we want to get the stdout from running the `hello` Script template. We have to assign the `Step` object
-returned from the function call to a variable, then we can access its `result` member, passing it to the `repeat_back`
-template.
+If we want to get the stdout from running the `hello` Script template, function call returns a `Step` object that we
```

Comment:
```suggestion
If we want to get the stdout from running the `hello` Script template, the function call returns a `Step` object that we
```

Reply:
Reworded/reorded this sentence, it was a bit clunky